kyo-tapir/shared/src/main/scala/kyo/Routes.scala

```diff
@@ -35,7 +35,7 @@ object Routes:
       *   A NettyKyoServerBinding wrapped in an asynchronous effect
       */
     def run[A, S](server: NettyKyoServer)(v: Unit < (Routes & S))(using Frame): NettyKyoServerBinding < (Async & S) =
-        Emit.run[Route].apply[Unit, Async & S](v).map { (routes, _) =>
```

Comment:
I think the explicit `apply` makes the code easier to follow. I wonder if we still need the explicit annotations